tests/test_actions.py

```diff
@@ -302,11 +305,11 @@ def test_mention_multiple_errors_once(self, mocked_time, Tester):
             Eventually(See(mock_question, IsEqualTo(False))).perform_as(Tester)
 
         assert str(actual_exception.value) == (
-            "Tester tried to Eventually see if returns bool is equal to <False> 3 times "
-            "over 20.0 seconds, but got:\n"
-            "    AssertionError: \n"
-            "Expected: <False>\n"
-            "     but: was <True>\n"
+            "Tester tried to Eventually see if returns bool is equal to <False> 3 times"
+            " over 20.0 seconds, but got:"
+            "\n    AssertionError: "
+            "\nExpected: <False>"
+            "\n     but: was <True>\n"
```

Comment:
This might be my own opinion, but i usually like to have the continuation characters at the start of the next line to make it clear how the continuation is joined on to the above line. 

I don't fully remember the reason i standardized on that. I think it just makes it easier to see at a glance, instead of having to follow the lines to the ends.

Reply:
Looks weird to me, but /shrug.  I've seen it enough both ways I'm not super bothered by it.